app/views/videos/show.html.haml

```diff
@@ -20,7 +25,7 @@
         %fieldset
           .row
             = f.alert_message "Please fix the following problems:", error_summary: false
-            = f.select :rating, [["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]], { label: "Rate this video" }, wrapper_class: 'col-sm-3'
+            = f.select :rating, ratings_choices, { label: "Rate this video" }, wrapper_class: 'col-sm-3'
```

Comment:
Thanks! I *may* have seen something about this in the forums. :wink:  Plus I know I'll need it for the queue form.